VEC-M-0080/main.tex

```diff
@@ -274,16 +258,16 @@ \subsection*{Cross Product and the Angle between Vectors}
 \section*{Practice Problems}
 \begin{problem}
 $\vec{i}\times\vec{k}=$
-\begin{multipleChoice}\choice{$\vec{j}$}
-  \choice[correct]{$-\vec{j}$ }
+\begin{multipleChoice}\choice{\vec{j}}
```

Comment:
It looks like you have to have the $ signs around the \vec{j} on these lines and the ones that follow.  Travis is complaining about \mathbf outside of math mode.

Reply:
okay